app/component/map/popups/DynamicParkingLotsPopup.js

```diff
@@ -35,6 +37,50 @@ class DynamicParkingLotsPopup extends React.Component {
     lon: PropTypes.number.isRequired,
   };
 
+  getPaidHours() {
+    const NOW = new Date();
+    const inSevenDays = Moment()
+      .add(7, 'days')
+      .format('YYYY/MM/DD');
+    const text = [];
+    let openUntil = '';
+
+    Moment.locale('de');
+
+    const paidHours = new OpeningHours(
+      this.props.feature.properties.paid_hours,
+      null,
+      { locale: 'en' },
+    );
```

Comment:
Is this the "import" locale or the "render/output" locale? And does it match the data?

Reply:
In this case it is the language of the error messages of the opening_hours module.

Reply:
What does this mean? Will the German user then see English error messages?

Reply:
The error messages go to the console, so I guess users won't see any. However I can configure this also to the context locale, if you'd like. I'll do it tomorrow.